openhands/llm/llm.py

```diff
@@ -184,7 +192,25 @@ def wrapper(*args, **kwargs):
 
             # log the response
             message_back = resp['choices'][0]['message']['content']
-            llm_response_logger.debug(message_back)
+
+            token_logs = 'Input tokens: ' + str(resp.usage.prompt_tokens) + '\n'
+            token_logs += 'Output tokens: ' + str(resp.usage.completion_tokens) + '\n'
+
+            if 'cache_creation_input_tokens' in resp.usage.model_extra:
+                token_logs += (
+                    'Input tokens (cache write): '
+                    + str(resp.usage.model_extra['cache_creation_input_tokens'])
+                    + '\n'
+                )
+
+            if 'cache_read_input_tokens' in resp.usage.model_extra:
+                token_logs += (
+                    'Input tokens (cache read): '
+                    + str(resp.usage.model_extra['cache_read_input_tokens'])
+                    + '\n'
+                )
+
+            llm_response_logger.debug(message_back + '\n\n\n' + token_logs)
```

Comment:
I think we could add some conditional for `usage` and its attributes, it's not clear to me that litellm response will always have it.

As a side note, what do you think about doing this in `_post_completion`, which is already doing the same kind of thing, logging costs? It will be displayed in console too then, and included in the regular logs. Not a big deal, though, just a thought.